source/options.tsx

```diff
@@ -71,6 +72,25 @@ async function init(): Promise<void> {
 		container.prepend(unchecked.closest('.feature')!);
 	}
 
+	// Highlight new features
+	const newFeatures: Record<string, number> = await cache.get<Record<string, number>>('features-new') ?? {};
+	if (browser.runtime.getManifest().version !== '19.12.22') { // First version introducing highlighting
+		const timespan = 10 * 24 * 60 * 60 * 1000; // Highlighting will be shown for 10 days after opening the options
```

Comment:
What if we make it clearer?
```suggestion
		const tenDaysAgo = Date.now() - (10 * 24 * 60 * 60 * 1000);
```

And then `newFeatures[feature.id] > tenDaysAgo, add class`